src/main/java/com/openelements/conduct/integration/fallback/FallbackConductChecker.java

```diff
@@ -0,0 +1,146 @@
+package com.openelements.conduct.integration.fallback;
+
+import com.openelements.conduct.data.CheckResult;
+import com.openelements.conduct.data.CodeOfConductProvider;
+import com.openelements.conduct.data.ConductChecker;
+import com.openelements.conduct.data.Message;
+import com.openelements.conduct.data.TextfileType;
+import com.openelements.conduct.data.ViolationState;
+import org.jspecify.annotations.NonNull;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+import java.util.regex.Pattern;
+
+public class FallbackConductChecker implements ConductChecker {
+
+    private static final Logger log = LoggerFactory.getLogger(FallbackConductChecker.class);
+    
+    private final CodeOfConductProvider codeOfConductProvider;
+
+    private static final Set<String> OFFENSIVE_TERMS = new HashSet<>(Arrays.asList(
+            "idiot", "stupid", "dumb", "moron", "retard", 
+            "crap", "shit", "fuck", "damn", "ass", "asshole",
+            "bitch", "bastard", "dick", "cunt", "whore", "slut",
+            "hate", "kill", "die", "attack", "terrible", "horrible",
+            "useless", "worthless", "incompetent", "pathetic"
+    ));
+
+    private static final Set<String> DISCRIMINATORY_TERMS = new HashSet<>(Arrays.asList(
+            "racist", "sexist", "homophobic", "transphobic", "bigot",
+            "nazi", "fascist", "communist", "terrorist",
+            "gay", "lesbian", "trans", "queer", "black", "white", "asian", "latino", "hispanic",
+            "jew", "muslim", "christian", "hindu", "buddhist",
+            "woman", "man", "girl", "boy", "female", "male"
```

Comment:
This class is way behind the scope of the project. If no AI api is integrated the application should just not work. 
Why?
Most terms that you have in the list can be used in a non-discrementing or non-offensive context. "The **attack** button currently has a **black** font color". A really meaningfull check will only work with understanding the context of the words and that is only doable by manual checks or AI.

Even with the addition of the attack pattern the software will only check a very limited range of possible code of conduct violations.

Based on that we should remove this class/functionality.